### DISCUSSION TITLE
[Projects Beta] Add new item on top instead of at the bottom

### DISCUSSION THREAD

**Discussion by robertocommit** [community/community#11082] [Projects and Issues]
I have a long list of items, and by default a new item is always added at the bottom.

I would love to have it at the top.

Many thanks

**Comment by alex-piccione** [comment]
It happens also when a Pull Request is merged and the task is automatically moved to the "Done" column.

**Comment by estellecomment** [comment]
We add new issues automatically to a board, so that we can see them and do something.
But they are added at the bottom, so we don't see them...

**Comment by julienfouilhe** [comment]
Same problem with workflows: when I close issues, I use a workflow to set the status to "Done", which has its own column in my Kanban project.
Issues are added to the bottom when I would like them added to the top so that I can see what was finished recently easily.

**Comment by aabouzaid** [comment]
Putting the items to the bottom of the column is super annoying because it's not visible and also needs a lot of scrolling.

It will be great to have a way to sort the items exactly like what we have in the Github discussions.

![Screenshot from 2022-11-09 10-56-07](https://user-images.githubusercontent.com/6760103/200799593-6195b14c-bfa8-4ada-b35d-b87b589e97bb.png)

**Comment by mohamedds-12** [comment]
Same problem, and it's so annoying!!!

**Comment by Disper** [comment]
<img width="267" alt="image" src="https://user-images.githubusercontent.com/1783008/235113329-c7b28d34-51c2-427e-8446-b46fe465ab2f.png">
It would be great if at least there would be a possibility to **Move the item to the top** in this context menu.

**Reply by Erkansukgen** [reply]
Go

**Comment by saumier** [comment]
Same problem. There should be a project or view based configuration to move cards to the top of the column.

**Comment by davidbarrowsatasos** [comment]
Also infinite scroll is a kind of curse in this situation.  The behaviour is jerky.  So if you have many items the workflow is - add an item (convenient); get the dialog for the blank issue (also a good UX); but then, a toast that says "the item has been added to the bottom of the column"; then, a minute of "infinite scrolling" where you go get the item, then say "move the item to the top of the column".  That bit is just silly.  If I'm trying to fix a bug and it suddenly becomes the top priority, I want that item on top by default, not the bottom.

**Comment by alexrosen** [comment]
I just moved from Jira. I had always heard it was clunky vs. GitHub. Then, I found this. It's an annoyance I experience multiple times per day. I can't believe people here have been raising it for over three years and it's not been addressed. It's so frustrating, I'll have to warn others against making the change to Projects.
